apps/files/src/components/Roles/RolesSelect.vue

```diff
@@ -3,8 +3,11 @@
     <label class="oc-label">
       <translate>Role:</translate>
     </label>
-    <oc-button :id="`files-${mode}-role-button`" :class="`uk-width-1-1 files-${mode}-role-button`">
-      <role-item :role="selectedRole" class="uk-margin-small-bottom" />
+    <oc-button
+      :id="`files-${mode}-role-button`"
+      :class="`uk-width-1-1 files-${mode}-role-button uk-flex-left`"
+    >
+      <role-item :role="selectedRole" class="uk-margin-xsmall-top uk-margin-xsmall-bottom" />
```

Comment:
Padding classes would be much better here but we don't have an xsmall ones 🙁

Reply:
Looks nice, let's keep it like this for now 👍